os/emscripten/pre.js

```diff
@@ -71,6 +71,31 @@ Module.preRun.push(function() {
          *  add_server("localhost", 3979); */
     }
 
+    var leftButtonDown = false;
+    document.addEventListener("mousedown", e => {
+	if(e.button == 0)
+            leftButtonDown = true;
```

Comment:
Some coding-style thingies:
```
if (e.button == 0) {
  leftButtonDown = true;
}
```

(space between `if` and `(`, always use `{}` if on more than one line)